src/Tempest/Auth/src/Install/UserPermission.php

```diff
@@ -14,6 +14,5 @@ final class UserPermission implements DatabaseModel
     public function __construct(
         public User $user,
         public Permission $permission,
-    ) {
-    }
```

Comment:
Most of the diff is regarding this change, we can disalbe this using the new `inline_empty_constructor_braces` option.

Reply:
This is actually a great change 👀